connector/google/google.go

```diff
@@ -46,16 +46,22 @@ func (c *Config) Open(logger logrus.FieldLogger) (conn connector.Connector, err
 	}
 	//creating client for extraxting groups for users
 	clientID := c.ClientID
-	contents, err := ioutil.ReadFile(c.filePath)
+	contents, err := ioutil.ReadFile(c.FilePath)
+	if err != nil {
+		fmt.Errorf("Unable to read file: %v", err)
```

Comment:
these error check blocks should return from the function instead of just continuing.